### PR TITLE
129 organization homepage

### PR DESCRIPTION
**Pull request by tanmaythole**
Organizations Homepage Layout - UI design

Tasks:
- [x] Create a redux store
- [X] Setup routing
- [X] UI design
- [ ] API integration

Ticket:
https://trello.com/c/T95SGVpX/127-invite-users-page-in-onboarding-organization

Screenshots:
Screenshot from 2023-01-05 12-49-41

### CODE REVIEW

File: src/store/Organizations/actions.js [hunk 1]
```diff
@@ -11,6 +11,20 @@ export const getOrganizationsSuccess = (data) => ({
 });
 
 export const getOrganizationsFailure = () => ({
+  type: orgConstants.GET_ORGANIZATION_FAILURE,
```
Comment (tanmaythole) [inline src/store/Organizations/actions.js]: for the `getOrganizationsFailure` action type should be `GET_ORGANIZATIONS_FAILURE`

File: src/store/Organizations/actions.js [hunk 2]
```diff
@@ -11,6 +11,20 @@ export const getOrganizationsSuccess = (data) => ({
...
+  payload: data,
+});
+
+export const getOrganizationSuccess = (data) => ({
+  type: orgConstants.GET_ORGANIZATION_SUCCESS,
+  payload: data,
+});
+
+export const getOrganizationFailure = () => ({
   type: orgConstants.GET_ORGANIZATIONS_FAILURE,
```
Comment (tanmaythole) [inline src/store/Organizations/actions.js]: for the `getOrganizationFailure` action type should be `GET_ORGANIZATION_FAILURE`

File: src/containers/OrganizationContainer/Organizations/index.jsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  const { org } = useParams();
+  const { organizationInfo } = useSelector(mapState);
+
+  const { name, description, image: imageSrc } = organizationInfo;
+
+  useEffect(() => {
+    org && dispatch(getOrganizationRequest({ org }));
+  }, [org]);
+
+  const activeRoute = buildRoute(ORG_OVERVIEW_ROUTE, { org });
```
Comment (tanmaythole) [inline src/containers/OrganizationContainer/Organizations/index.jsx]: add comment

File: src/Components/OrganizationsComponents/Organization/OrgNavItem.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        </span>
+      </Nav.Link>
+    </Nav.Item>
+  );
+};
+
+OrgNavItem.propTypes = {
+  data: PropTypes.shape({
+    title: PropTypes.string.isRequired,
+    icon: PropTypes.element.isRequired,
```
Comment (tanmaythole) [inline src/Components/OrganizationsComponents/Organization/OrgNavItem.jsx]: check whether the propTypes for icon is correct

File: src/store/Organizations/actions.js (hunk 1 above)
Comment (tanmaythole) [reply src/store/Organizations/actions.js]: 👍🏻

File: src/store/Organizations/actions.js (hunk 2 above)
Comment (tanmaythole) [reply src/store/Organizations/actions.js]: done 👍🏻

File: src/containers/OrganizationContainer/Organizations/index.jsx (hunk 3 above)
Comment (tanmaythole) [reply src/containers/OrganizationContainer/Organizations/index.jsx]: done 👍🏻

File: src/Components/OrganizationsComponents/Organization/OrgNavItem.jsx (hunk 4 above)
Comment (tanmaythole) [reply src/Components/OrganizationsComponents/Organization/OrgNavItem.jsx]: done 👍🏻

File: src/Components/OrganizationsComponents/Organization/OrgNavItem.jsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { Nav } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import PropTypes from "prop-types";
+
+import { buildRoute } from "utils/helper";
+
+const OrgNavItem = ({ data, org }) => {
+  const { title, icon: Icon, url } = data;
```
Comment (ChinmaySonawane) [inline src/Components/OrganizationsComponents/Organization/OrgNavItem.jsx:9]: why we are passing Icon as props?
Is it component or argument?

File: src/store/Organizations/apis.js [hunk 6]
```diff
@@ -1,3 +1,4 @@
+/* eslint-disable no-unused-vars */
```
Comment (ChinmaySonawane) [inline src/store/Organizations/apis.js]: Why to disable whole file for single variable?
instead on that we can use `eslint-disable-next-line  no-unused-vars`

Comment (tanmaythole) [reply src/store/Organizations/apis.js]: 👍🏻

**Review by ChinmaySonawane — APPROVED** [review]

**Pull request merged**